docs/modules/vertiq_23xx_family.rst

```diff
@@ -0,0 +1,29 @@
+
+.. _vertiq_23xx_family:
+
+*********************************
+Vertiq 23-XX Family 
+*********************************
+
+.. image:: ../_static/module_pictures/23xx_family.png
+        :alt: Vertiq 23-XX Family
+        :width: 500
+        :align: center
+
+.. csv-table:: Vertiq 23-XX Family of Modules
+        :header: "Size", "Kv", "Default Firmware", "Available Firmware"
+        :align: center
+
+        "23-06", "220", "Servo", "Servo"
+        "23-06", "2200", "Speed", "Speed, Pulsing"
+        "23-14", "920", "Speed", "Servo, Speed"
```

Comment:
We've never actually had a servo firmware for the 2314 to the best of my knowledge. I don't know that we plan on doing one at the moment,